common/static/common/js/discussion/views/discussion_thread_list_view.js

```diff
@@ -731,18 +755,25 @@
                     url: DiscussionUtil.urlFor("users"),
                     type: "GET",
                     dataType: 'json',
-                    error: function() {
-                    },
+                    error: function() {},
                     success: function(response) {
-                        var message;
+                        var message, username;
                         if (response.users.length > 0) {
-                            message = interpolate(_.escape(gettext('Show posts by %(username)s.')), {
-                                "username": _.template('<a class="link-jump" href="<%= url %>"><%- username %></a>')({
-                                    url: DiscussionUtil.urlFor("user_profile", response.users[0].id),
-                                    username: response.users[0].username
-                                })
-                            }, true);
-                            return self.addSearchAlert(message);
+                            username = edx.HtmlUtils.joinHtml(
+                                edx.HtmlUtils.HTML(
+                                    edx.StringUtils.interpolate(
+                                        '<a class="link-jump" href="{url}">',
+                                        {url: DiscussionUtil.urlFor("user_profile", response.users[0].id)}
+                                    )
+                                ),
```

Comment:
The url should be escaped, so you want something like (untested):
```
edx.HtmlUtils.joinHtml(
    edx.HtmlUtils.HTML('<a class="link-jump" href="'),
    DiscussionUtil.urlFor("user_profile", response.users[0].id),
     edx.HtmlUtils.HTML('">')
)
```
Or you could use `edx.HtmlUtils.interpolateHtml()` and wrap the first param with `HtmlUtils.HTML()` and interpolate in the url as you did above.